Classes/ViewHelpers/FormViewHelper.php

```diff
@@ -268,7 +274,9 @@ protected function renderTrustedPropertiesField()
      */
     protected function renderFormUidField()
     {
-        $this->addHiddenField($this->prefixFieldName('mail[form]'), $this->arguments['formUid']);
+        $fieldName = $this->prefixFieldName('mail[form]');
+        $this->registerFieldNameForFormTokenGeneration($fieldName);
+        $this->addHiddenField($this->prefixFieldName($fieldName), $this->arguments['formUid']);
```

Comment:
I have tested your refactoring on a recently to LTS12 upgraded project and found `$this-prefixFieldName` here seems "doubled" and produces `tx_powermail_pi1[tx_powermail_pi1][mail][form]` but it should be `tx_powermail_pi1[mail][form]`

Reply:
@nabossha, good catch! Thanks for pointing out. It's fixed now :muscle: